tensorflow/tools/ci_build/ci_parameterized_build.sh

```diff
@@ -501,14 +501,21 @@ fi
 
 chmod +x ${TMP_SCRIPT}
 
+# Map TF_BUILD container types to containers we actually have.
+if [[ "${CTYPE}" == "android_nightly" ]]; then
```

Comment:
My nit point is the word "nightly" doesn't really tell us what the more extensive build does. It is also less flexible lest we decide to run the build in some cadence other than nightly (e.g., hourly or postsubmit).